src/discussions/posts/post/PostLink.jsx

```diff
@@ -24,12 +26,12 @@ function PostLink({
       }
     >
       {post.abuseFlagged && (
-        <div className="bg-gray-100 flex-fill">
-          <FontAwesomeIcon icon={faFlag} className="mx-1" />
-          {intl.formatMessage(messages.contentReported)}
+        <div className="align-items-center bg-danger-100 d-flex flex-fill p-1">
+          <Icon className="text-danger-700" src={Flag} />
+          <span className="text-gray-700 x-small">{intl.formatMessage(messages.contentReported)}</span>
         </div>
       )}
-      <div className="d-flex flex-row mb-2">
+      <div className="d-flex flex-row">
         <FontAwesomeIcon icon={faCircle} className={`my-1 text-accent-a ${post.read && 'invisible'}`} size="xs" />
```

Comment:
is that Icon missing in paragon?

Reply:
Yes, I forgot to mention that; `faCircle` and `faQuestionCircle` were missing last time I checked.

Reply:
Okay will add that in paragon for you.

Reply:
can you please provide the Figma link where you need fa circle icon. because there is confusion in the team that no design is using the fa-circle icon. and also on the attached screenshot, there is no fa-circle icon is visible

Reply:
Sure: here's [the figma link](https://www.figma.com/file/LVFnO9oRHYiamoicZuXDAC/Discussion-page?node-id=1918%3A53114); you'll notice there's a blue circle on the some of the posts, I'm guessing it indicates they're unread.

I *think* the logic for marking posts as read/unread isn't implemented yet, which is why it doesn't appear.

Reply:
and I am not sure why we need an icon to display the indicator it can be achieved with the CSS provided in the mockups.

Reply:
Done, I've updated the screenshot with how it looks.

Reply:
@kewne since it's not merged and I have an update from UX they provided me with the icon svg and I have added the icon in paragon with the name **Unread** you can use it as well.